x-pack/plugins/triggers_actions_ui/public/application/sections/rules_list/components/rules_list.tsx

```diff
@@ -886,7 +887,7 @@ export const RulesList = ({
         onPage={setPage}
         onRuleChanged={() => refreshRules()}
         onRuleClick={(rule) => {
-          const detailsRoute = ruleDetailsRoute ? ruleDetailsRoute : routeToRuleDetails;
+          const detailsRoute = ruleDetailsRoute ? ruleDetailsRoute : commonRuleDetailsRoute;
           history.push(detailsRoute.replace(`:ruleId`, rule.id));
```

Comment:
Is there some reason we don't use `getRuleDetailsRoute(rule.id)` here, as was done in `rule_details.tsx` above?

Reply:
The `ruleDetailsRoute` is passed in as a prop to this component and it wasn't clear to me what the value would be. So I don't think we can rely on it equaling `commonRuleDetailsRoute` aka `'/rule/:ruleId'`. So if we called `getRuleDetailsRoute` here it could change the initial part of the path (`/rule` may not exist in the `ruleDetailsRoute` passed in as a prop).

Another option is that we change `getRuleDetailsRoute` to take the path portion but that's pretty close to just calling `.replace` directly 🤷‍♂️